src/Mgate/PersonneBundle/Entity/PersonneRepository.php

```diff
@@ -105,15 +105,22 @@ public function getMembreNotUser($user = null)
 
     /**
      * Requete sur l'ensemble des personnes avec en jointure les différents OneToOne possibles.
+     *
+     * @param bool $orderByNom
+     *
+     * @return mixed
      */
-    public function getAllPersonne()
+    public function getAllPersonne($orderByNom = false)
```

Comment:
you might as well add the typehint `bool` or for new stuff I would recommend to use more sensible names, i.e. `$orderByName` here.

The return type `mixed` also looks weird, it should be `: array` with the phpdoc block complementing it to `Personne[]` IMO